<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_modi_1</class>
 <widget class="QDialog" name="fct_modi_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des inscriptions / désinscriptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_fct_modi_1_inscription">
     <property name="title">
      <string>Insérer des résultats</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="vl_insert_1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="epreuve">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Epreuve</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_epreuve">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_insert_2">
        <item>
         <widget class="QLabel" name="medailleOr">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Médaille d'Or</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_gold"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_insert_3">
        <item>
         <widget class="QLabel" name="medailleArgent">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Médaille d'Argent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_silver"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_insert_4">
        <item>
         <widget class="QLabel" name="medailleBronze">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Médaille de Bronze</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_bronze"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_insert_5">
        <item>
         <widget class="QPushButton" name="pushButton_insert">
          <property name="text">
           <string>Insérer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_modi">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_modi_2">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
